input/ignoreWarnings.txt

```diff
@@ -4,3 +4,7 @@
 The HTML fragment 'ip-statements.xhtml' is not included anywhere in the produced implementation guide
```

Comment:
Commenting here to get an inline comments. 

This PR does not include changes to ignoreWarnings or changes files, please remove from the PR.